sql/migrations/20220404185055_world.sql

```diff
@@ -0,0 +1,74 @@
+(566101, 30, 0, 15, 7162, 0, 0, 0, 31894, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Cast Spell'),
+(566101, 33, 0, 35, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3.9619, 0, 'Brother Malach - Set Orientation'),
+(566101, 34, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1980, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Talk'),
+(566101, 37, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 8306, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Talk'),
+(566101, 39, 0, 10, 5685, 0, 0, 0, 0, 0, 0, 0, 0, 568001, 0, 2, 1735.24, 380.689, -62.2077, 3.85718, 0, 'Brother Malach - Summon Creature'),
+(566101, 39, 0, 10, 5686, 0, 0, 0, 0, 0, 0, 0, 0, 568001, 0, 2, 1734.68, 378.533, -62.1315, 3.85718, 0, 'Brother Malach - Summon Creature'),
+(566101, 56, 0, 35, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3.88974, 0, 'Brother Malach - Set Orientation'),
+(566101, 57, 0, 15, 7162, 0, 0, 0, 38130, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Cast Spell'),
+(566101, 63, 0, 35, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4.26256, 0, 'Brother Malach - Set Orientation'),
+(566101, 64, 0, 15, 7162, 0, 0, 0, 31894, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Cast Spell'),
+(566101, 67, 0, 35, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3.9619, 0, 'Brother Malach - Set Orientation'),
+(566101, 68, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1984, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Talk'),
+(566101, 71, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 8307, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Talk'),
+(566101, 74, 0, 10, 5687, 0, 0, 0, 0, 0, 0, 0, 0, 568001, 0, 2, 1735.33, 379.152, -62.206, 3.82227, 0, 'Brother Malach - Summon Creature'),
+(566101, 129, 0, 35, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3.88974, 0, 'Brother Malach - Set Orientation'),
+(566101, 130, 0, 15, 7162, 0, 0, 0, 38130, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Cast Spell'),
+(566101, 134, 0, 35, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4.26256, 0, 'Brother Malach - Set Orientation'),
+(566101, 135, 0, 15, 7162, 0, 0, 0, 31894, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Cast Spell'),
+(566101, 138, 0, 35, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3.9619, 0, 'Brother Malach - Set Orientation'),
+(566101, 139, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1985, 0, 0, 0, 0, 0, 0, 0, 0, 'Brother Malach - Talk');
+INSERT INTO `generic_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(568001, 0, 0, 15, 7791, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive - Cast Spell'),
+(568001, 0, 0, 15, 7791, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Captive - Cast Spell');
+
+-- NOTE: Text ID 1983 'You show some worth after all. Lysta summon a surprise for our young recruits.' Exists in the DB But I have never seen it used in the event? Perhaps Something triggers an additional part of the event?

```

Comment:
We can't forget classic does not have all the little events implemented. Some were bugged and cut away from the game.
However this could be part of a quest.